config/routes.rb

```diff
@@ -84,6 +84,14 @@
   match '/admin/newsletter' => 'admin#newsletter', :as => :admin_newsletter
   match '/admin/:action' => 'admin#:action'
 
-
+  namespace :api, :defaults => {:format => :json}  do
+    namespace :v1 do
+      resources :crops
+      resources :gardens
```

Comment:
I have other files for the other main content types, once crops is looking better would it make sense to enable the endpoints for those at a very basic level too?